service/key/key.go

```diff
@@ -35,6 +35,10 @@ func BucketName(customObject awstpr.CustomObject, accountID string) string {
 	return fmt.Sprintf("%s-g8s-%s", ClusterID(customObject), accountID)
 }
 
+func BucketObjectName(customObject awstpr.CustomObject, prefix string) string {
+	return fmt.Sprintf("cloudconfig/%s/%s", ClusterVersion(customObject), prefix)
+}
```

Comment:
Nice, I think this is the correct key structure. We can look at S3 versioning later if necessary.